bingo.js

```diff
@@ -435,6 +451,21 @@ function changeColourCount(value)
 	pushNewLocalSetting(COLOUR_COUNT_SETTING_NAME, COLOURCOUNT);
 }
 
+function updateColourSymbols()
+{
+	$(".symbol").css("display", !COLOURSYMBOLS ? "none" : "inline");
+	const button = document.getElementById("colourSymbols");
+	button.innerHTML = COLOURSYMBOLS ? "Hide Symbols" : "Show Symbols";
+}
+
+function toggleColourSymbols(value)
+{
+	// Invert HIDDEN setting, then update
```

Comment:
I've just noticed that this comment should be updated after copy-pasting from `toggleHidden()`.